internal/commands/run.go

```diff
@@ -194,6 +208,8 @@ func newRunCmd() *cobra.Command {
 	cmd.PersistentFlags().StringArrayVarP(&environment, "environment", "e", []string{},
 		"set (-e A=B) or pass-through (-e A) an environment variable")
 	cmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "")
+	cmd.PersistentFlags().StringVarP(&output, "output", "o", "auto", "output format of logs, "+
```

Comment:
I think we need to show the default value here.

Reply:
Should be fine, cobra shows it.

```bash
$: cirrus run --help
...
  -o, --output string             output format of logs, supported values: auto, interactive, simple, travis, github-actions (default "auto")
...
```